core/Enclave/enclave_hsm.cpp

```diff
@@ -210,7 +211,8 @@ sgx_status_t enclave_get_public_key(ehsm_keyblob_t *cmk, size_t cmk_size,
     if (cmk == NULL ||
         cmk_size != APPEND_SIZE_TO_KEYBLOB_T(cmk->keybloblen) ||
         cmk->keybloblen == 0 ||
-        cmk->metadata.origin != EH_INTERNAL_KEY)
+        cmk->metadata.origin != EH_INTERNAL_KEY ||
+        cmk->metadata.keyusage == NULL)
```

Comment:
keyusage is a enum type, why compare with NULL?

Reply:
fix

Reply:
not fixed?